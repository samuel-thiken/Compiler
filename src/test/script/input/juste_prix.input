1
50
25
12
7
9
10