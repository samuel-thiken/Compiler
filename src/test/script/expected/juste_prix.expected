Bienvenue au juste prix !
Veuillez entrer le nombre de parties que vous voulez faire :
Entrez un nombre entre 1 et 100...
Entrez un nombre entre 1 et 100...
Plus petit !
Entrez un nombre entre 1 et 100...
Plus petit !
Entrez un nombre entre 1 et 100...
Plus petit !
Entrez un nombre entre 1 et 100...
Plus grand !
Entrez un nombre entre 1 et 100...
Plus grand !
Bravo, le nombre Ã©tait bien 10 !
